src/QsCompiler/CommandLineTool/CommandLineTool.csproj

```diff
@@ -33,7 +33,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="!$([MSBuild]::IsOsPlatform('Windows'))">
-    <PackageReference Include="Microsoft.Quantum.LlvmBindings.Native" Version="13.0.0-CI-20220129-005156">
+    <PackageReference Include="Microsoft.Quantum.LlvmBindings.Native" Version="13.0.0-CI-20221216-194411">
```

Comment:
Same comment re some LLVM versions being 13.x and 14.x, which looks a little odd.

Reply:
Should be fixed in main... I'll get a PR open for that.

Reply:
Fixing in https://github.com/microsoft/qsharp-compiler/pull/1593